python/pyarrow/parquet.py

```diff
@@ -555,7 +550,7 @@ def __init__(self, where, schema, filesystem=None,
         else:
             sink = where
         self._metadata_collector = options.pop('metadata_collector', None)
-        engine_version = os.environ.get('ARROW_PARQUET_WRITER_ENGINE', 'V2')
```

Comment:
There is another occurrence of this in ``_dataset.pyx`` now

Reply:
thanks.  removed.